homeassistant/components/wemo/light.py

```diff
@@ -175,21 +175,27 @@ def turn_on(self, **kwargs):
             if color_temp is not None:
                 self.wemo.set_temperature(mireds=color_temp, transition=transition_time)
 
-            self.wemo.turn_on(**turn_on_kwargs)
+            if self.wemo.turn_on(**turn_on_kwargs):
+                self._state = 1
```

Comment:
`self._state` should be a dictionary it seems.

Reply:
It's just a binary on = 1, off = 0.

Reply:
I can make it a dict, if needed.

Reply:
Not for the `WemoLight` entity:
https://github.com/home-assistant/core/blob/02f174e2e6abeac00cef840590b38a93d43d88fc/homeassistant/components/wemo/light.py#L193-L208

Reply:
Good catch. Thank you.

Reply:
I apologize about the dict as well. I realize I had constants defined for on/off already, and forgot to use them here. I'm fixing that and the WemoLight as well, and will push some changes once I test them locally.